app/controllers/rooms_controller.rb

```diff
@@ -1,29 +1,46 @@
 class RoomsController < ApplicationController
-  before_action :load_rooms, except: %i(show)
+  before_action :load_rooms, only: %i(index)
   before_action :load_room, only: %i(show)
+  before_action :load_attributes
 
-  def index
-    @attributes = RoomAttribute.pluck(:name, :id)
-  end
+  # routes
+
+  def index; end
 
   def show; end
 
   private
 
+  # before action
+
+  def load_attributes
+    @attributes = RoomAttribute.pluck(:name, :id)
+  end
+
   def load_room
-    @room = Room.find_by id: params[:id]
-    return if @room
+    @room = Room.find_by id: prs[:id]
+    return if @room.any?
```

Comment:
```suggestion
    return if @room
```
ri là sao em? 
sao lại dùng `any?`

Reply:
Em đã fix ạ